common/autoware_lanelet2_ros_interface/include/autoware_lanelet2_ros_interface/utility/internal/query.tpp

```diff
@@ -51,6 +62,28 @@ struct RecurseVisitor : public RuleParameterVisitor {
   query::References &rfs_;
 };
 
+// Helper visitor class for finding existing elements with same id in the map and assigning input to it
+struct OverwriteParameterVisitor : public lanelet::internal::TrueMutableParameterVisitor {
+  explicit OverwriteParameterVisitor (lanelet::LaneletMapPtr ll_Map) : ll_Map_(ll_Map) {}
+  void operator() (Point3d& p) override { overwriteWithMatchingId(p, ll_Map_);} 
+  void operator() (LineString3d& ls) override { overwriteWithMatchingId(ls, ll_Map_);}
+  void operator() (Polygon3d& poly) override { overwriteWithMatchingId(poly, ll_Map_);}
+  void operator() (WeakLanelet& llt) override { 
+    if (llt.expired()) {  // NOLINT
+      return;
```

Comment:
Would a debug log be valuable here? Or is this an expected occurrence? (same below)

Reply:
Added